cli/parser/bazelrc/bazelrc.go

```diff
@@ -297,12 +265,28 @@ func GetBazelOS() string {
 	}
 }
 
+// IsBazelCommand returns whether the given string is recognized as a bazel
+// command.
 func IsBazelCommand(command string) bool {
 	_, ok := bazelCommands[command]
 	return ok
 }
 
+// Parent returns the parent command of the given command, if one exists.
 func Parent(command string) (string, bool) {
 	parent, ok := parentCommand[command]
 	return parent, ok
 }
+
+// IsCommonPhase returns whether or not this is a phase that should always
+// be evaluated, regardless of the command.
+func IsCommonPhase(phase string) bool {
+	_, ok := commonPhases[phase]
+	return ok
+}
```

Comment:
I would maybe call this "IsUnconditionalPhase" or "IsGlobalPhase" to avoid confusion with the phase that is named "common"

Reply:
Renamed to `IsUnconditionalCommandPhase` (since it only gets used for command options, not startup options)